src/codegen/spirv.zig

```diff
@@ -290,9 +290,6 @@ pub const DeclGen = struct {
     fn backingIntBits(self: *DeclGen, bits: u16) ?u16 {
         const target = self.getTarget();
 
-        // The backend will never be asked to compiler a 0-bit integer, so we won't have to handle those in this function.
-        assert(bits != 0);
-
```

Comment:
Did you run into a case where there was a 0 bit int?

Reply:
enums with one member are backed by `u0`. e.g. [this](https://github.com/ziglang/zig/pull/15748/files#diff-b445d3786cb03ccb30bab7e1a4babe9f4d54d8f083b48e6ebcbbaabf80d5b82aR843) or [this](https://github.com/ziglang/zig/pull/15748/files#diff-b445d3786cb03ccb30bab7e1a4babe9f4d54d8f083b48e6ebcbbaabf80d5b82aR847)

Reply:
But should those not be catched by `if (layout.tag_size == 0) return null;`? Operations on `u0` should never be generated, hence implementing arithmeticTypeInfo for those makes no sense